torchrl/envs/transforms/transforms.py

```diff
@@ -1584,15 +1584,16 @@ def __init__(
         super(CatTensors, self).__init__(
             keys_in=sorted(list(self.keys_in)), keys_out=[out_key]
         )
-        if (
-            ("reward" in self.keys_in)
-            or ("action" in self.keys_in)
-            or ("reward" in self.keys_in)
-        ):
-            raise RuntimeError(
-                "Concatenating observations and reward / action / done state "
-                "is not allowed."
-            )
+        # TODO: check this: do we want to allow it?
```

Comment:
I think that for now we can remove this
The idea was originally that it would be dangereous to concatenate reward and action for instance, but upon reflection i guess that if someone asks for that either she knows what she's doing or she's just far of understanding anything of this class. So I think we can remove that block